run_tests.sh

```diff
@@ -45,6 +53,88 @@ Extra options:
  --dockerize           Run tests in a pre-configured Docker container (must be first argument if present).
  --db <type>           For use with --dockerize, run tests using partially migrated 'postgres', 'mysql',
                        or 'sqlite' databases.
+  --external_url       External URL to use for Galaxy testing (only certain tests).
+  --external_master_key Master API key used to configure external tests.
+  --external_user_key  User API used for external tests - not required if
+                       external_master_key is specified.
+
+Environment Variables:
+
+In addition to the above command-line options, many environment Variables
+can be used to control the Galaxy functional testing processing.
```

Comment:
We should probably state that command-line options take precedence over environment variables (even if this is common practice).